src/css/base/_normalize.scss

```diff
@@ -0,0 +1,35 @@
+select,
```

Comment:
Why do we need this? Afaik Bootstrap already provides "Reboot". Isn't that more or less the same?